### ISSUE TITLE
Reactions take time until seen by others

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/spreed#7064]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. User A write a message
2. User B react to the message

### Expected behaviour
User A sees the reaction

### Actual behaviour
User A sees the reaction only after the 0-30s of the chat polling pass. This is some what a performance optimisation, but it can be totally unexpected in this case. But if otherwise 70 people react 70 people reload the chat 70 times => 4.9k requests...
Might not be solvable for now, but want this to be documented at least

## Talk app

**Talk app version:** master

**Comment by SystemKeeper** [comment]
Similar to typing indicator we could send a signaling message to the active participants in the room when adding/removing a reaction. Limiting it to external signaling should have no big performance issue and would not increase the load on the NC side

**Comment by nickvergessen** [comment]
Yeah can also just close it and it comes when all chat messages come via HPB

**Comment by Antreesy** [comment]
chat-relay arrives in Talk 23, so considered solved
